templates/lune/data.json

```diff
@@ -0,0 +1,44 @@
+{
+  "lune": {
+    "head-tag": {
+      "extensions": [
+        "amp-sidebar"
+      ],
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Lato:400,700,900"
+      ]
+    },
+    "navbar": {
+      "name": "AMP Lune template",
+      "sidebar-id": "header-sidebar",
+      "sidebaronly": true,
+      "href": "lune.amp.html",
+      "logo": {
+         "name": "LUNE",
+         "classes": "h3 xs-hide"
+      },
+      "fixeditems": [
+        {
+          "link": {
+            "url": "mailto:contact@lune.com",
+            "text": "contact@l-u-n-e.com"
+          }
+        }
+      ],
+      "nav": [
+        {
+          "link": {"url": "product-blanc.amp.html", "text": "<span>B</span><span>l</span><span>a</span><span>n</span><span>c</span>"}
```

Comment:
After looking at `amp-fit-text` I don't think it's going to give us the effect that we need. Unfortunately it looks like the component only adjusts the font size to fill the space, which is not the effect that we need. Ideally it would adjust the letter spacing to spread the word out. I can open up a feature request on the `amp-fit-text` component, but I'm not sure if it's too much of a bespoke case to add it in.